rest-api-spec/src/main/resources/rest-api-spec/test/search/360_number_field_term_doc_values_only.yml

```
@@ -0,0 +1,169 @@
+setup:
+    - skip:
+        features: [ "headers" ]
+
+    - skip:
+        version: " - 2.11.99"
+        reason: "searching with only doc_values was added in 2.12.0"
+---
+"search on number fields with doc_values enabled":
+  - do:
+      indices.create:
+        index:  test-iodvq
+        body:
+          mappings:
+            dynamic: false
+            properties:
+              byte:
+                type: byte
+                index: false
+                doc_values: true
+              double:
+                type: double
+                index: false
+                doc_values: true
+              float:
+                type: float
+                index: false
+                doc_values: true
+              half_float:
+                type: half_float
+                index: false
+                doc_values: true
+              integer:
+                type: integer
+                index: false
+                doc_values: true
+              long:
+                type: long
+                index: false
+                doc_values: true
+              short:
+                type: short
+                index: false
+                doc_values: true
+              unsigned_long:
+                type: unsigned_long
+                index: false
+                doc_values: true
+
+
+  - do:
+      headers:
+        Content-Type: application/json
+      index:
+        index:  "test-iodvq"
+        id:     1
+        body:
+          byte: 1
+          double: 1.0
+          float: 1.0
+          half_float: 1.0
+          integer: 1
+          long: 1
+          short: 1
+          unsigned_long: 1
```

Similarly to other test case, please:

```suggestion
          unsigned_long: 10223372036854775807
```